src/main/resources/site/filters/router.es6

```diff
@@ -31,7 +32,7 @@ exports.filter = function(req, next) {
     targetId = get({
       key: '/ssb/kommunefakta/kommune'
     })._id
-    pageTitle = `Kommunefakta ${region}`
+    pageTitle = `Kommunefakta ${region.charAt(0).toUpperCase() + region.slice(1)}`
```

Comment:
How about doing it in CSS ?

```CSS
.municipalityTitle {
  text-transform: capitalize;
}
```

```javascript
render() {
  pageTitle = (
    <span className="municipalityTitle">
      Kommunefakta {region.toLowerCase()}
    </span>
  )
}
```

Reply:
This is the <title> tag on the page, not an h-tag

Reply:
ah! indeed! :)